madlibs_list.txt

```diff
@@ -8,12 +8,12 @@
 8. Noun (Plural): Tigers
 9. Noun : Pink
 10. Room Name : A101
-11. Liquid
-12. Adjective
-13. Material
-14. Activity
-15. Adverb
-16. Number
+11. Liquid - blood
+12. Adjective - comfortable
+13. Material - mushrooms
+14. Activity - reading
+15. Adverb - murder mysteries
+16. Number - 1000
```

Comment:
16th blank is already handled and value exists in main branch. Please modify accordingly.